### DISCUSSION TITLE
LFO's improve

### DISCUSSION THREAD

**Discussion by javegon** [bangcorrupt/hacktribe#229] [Ideas]
I watch a video which shows different and great LFO's. Far from classics  Sine, Triangle...
Device like novation circuit has lfo like :
Patterns -  different types patterns
Scales -  Major, Minor, and more 
ARPs -Arpeggios 

It would be possible on electribe devices ?
I leave a link to show how it works. https://www.youtube.com/watch?v=3orEPK4h5yo

Thanks!

**Comment by bangcorrupt** [comment]
I haven't looked into the LFO's much yet.  There is a Sine LFO hidden, and we will be able to change parameters like beat sync and phase in the future.

**Reply by javegon** [reply]
thanks
